python/pyarrow/tests/test_table.py

```diff
@@ -3523,7 +3538,7 @@ def test_chunked_array_non_cpu(cuda_context, cpu_chunked_array, cuda_chunked_arr
 
     # filter() test
     with pytest.raises(NotImplementedError):
-        cuda_chunked_array.filter([True, False, True, False, True])
+        cuda_chunked_array.filter([True])
```

Comment:
Is there a reason for this change? (I would expect an error about a mismatch in mask length, and while we check for CPU first, it might be safe to ensure the argument is valid otherwise)

Reply:
Oof, I was trying to clean up changes and I think messed this up. Thanks for catching.